src/3id/keyring.js

```diff
@@ -58,7 +64,12 @@ class Keyring {
   }
 
   getDBKey () {
-    return this.signingKey.privateKey.slice(2)
+    return ec.keyFromPrivate(this.signingKey.privateKey.slice(2))
+  }
+
+  async getIdentity () {
```

Comment:
This would probably make more sense to have in the ThreeId class. That might need some refactoring though. Main reason is that the keyring will not know of the DID of the space.

Reply:
yeah makes senes,  this just to get things working, mostly waiting for 3id changes to see exactly how this would look and where, can probably start that now